API/NoDocs/Microsoft.Extensions.Diagnostics.HealthChecks/ResourceUtilizationHealthCheckOptions.cs

```diff
@@ -0,0 +1,19 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization'
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Options;
+using Microsoft.Shared.Data.Validation;
+
+namespace Microsoft.Extensions.Diagnostics.HealthChecks;
+
+public class ResourceUtilizationHealthCheckOptions
+{
+    [ValidateObjectMembers]
+    public ResourceUsageThresholds CpuThresholds { get; set; }
+    [ValidateObjectMembers]
+    public ResourceUsageThresholds MemoryThresholds { get; set; }
+    [Microsoft.Shared.Data.Validation.TimeSpan(100, int.MaxValue)]
```

Comment:
I know it's an internal type so it doesn't matter but it might  be nice to call TimeSpanAttribute something different. I assume collision with regular TimeSpan is why you are fully qualifying the namespace here.

Reply:
This is probably just an artifact of the tool that produces the API snapshots.

There is no conflict with the TimeSpan type. The compiler is smart enough to know what's going on.